code/game/objects/effects/decals/cleanable/misc.dm

```diff
@@ -25,6 +25,11 @@
 	. = ..()
 	reagents.add_reagent("ash", 30) //double the amount of ash.
 
+/obj/effect/decal/cleanable/glass
+	name = "tiny shards"
+	desc = "Back to sand."
+	icon = 'icons/obj/shards.dmi'
+	icon_state = "tiny"

```

Comment:
So they're tiny shards that you can mop up, or remove with space cleaner?

Reply:
Correct